ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/e2e/casework-regression/create-case-regions-group.cy.ts

```diff
@@ -108,56 +118,164 @@ describe("Creating a case", () => {
+		Logger.Log("Editing the existing region");
+		caseManagementPage.editManagedBy();
 
-        editRegionPage
-            .hasRegion("London")
-            .withRegion("South West")
-            .apply();
+		editRegionPage
+			.hasRegion("London")
+			.withRegion("South West")
+			.apply();
 
-        caseManagementPage.hasManagedBy("Regions Group", "South West");
+		caseManagementPage.hasManagedBy("Regions Group", "South West");
 
-        Logger.Log("Add another concern after case creation");
+		Logger.Log("Add another concern after case creation");
 		caseManagementPage.addAnotherConcern();
 
-        createConcernPage
-            .withConcernType("Safeguarding")
-            .withConcernRating("Red-Amber")
-            .withMeansOfReferral(SourceOfConcernExternal)
-            .addConcern();
+		createConcernPage
+			.withConcernType("Safeguarding")
+			.withConcernRating("Red-Amber")
+			.withMeansOfReferral(SourceOfConcernExternal)
+			.addConcern();
 
-        caseManagementPage
+		caseManagementPage
 			.hasConcerns("Governance", ["Amber", "Green"])
 			.hasConcerns("Safeguarding", ["Red", "Amber"]);
 
-        Logger.Log("Close down the concerns");
-        caseManagementPage
-            .getEditConcern().first().click();
-        
-        editConcernPage.closeConcern().confirmCloseConcern();
-
-        caseManagementPage
-        .getEditConcern().first().click();
-    
-        editConcernPage.closeConcern().confirmCloseConcern();
-
-        caseManagementPage.getCaseIDText()
-        .then((caseId =>
-        {
-            caseManagementPage.getCloseCaseBtn().click();
-            caseManagementPage.withRationaleForClosure("Closing").getCloseCaseBtn().click();
-    
-            Logger.Log("Viewing case is closed");
-            homePage.getClosedCasesBtn().click();
-    
-            Logger.Log("Checking accessibility on closed case");
-            cy.excuteAccessibilityTests();
-    
-            closedCasePage.getClosedCase(caseId).click();
-    
-            viewClosedCasePage.hasManagedBy("Regions Group", "South West");
-        }));
+		Logger.Log("Close down the concerns");
+		caseManagementPage
+			.getEditConcern().first().click();
+
+		editConcernPage.closeConcern().confirmCloseConcern();
+
+		caseManagementPage
+			.getEditConcern().first().click();
+
+		editConcernPage.closeConcern().confirmCloseConcern();
+
+		caseManagementPage.getCaseIDText()
+			.then((caseId => {
+				caseManagementPage.getCloseCaseBtn().click();
+				caseManagementPage.withRationaleForClosure("Closing").getCloseCaseBtn().click();
+
+				Logger.Log("Viewing case is closed");
+				homePage.getClosedCasesBtn().click();
+
+				Logger.Log("Checking accessibility on closed case");
+				cy.excuteAccessibilityTests();
+
+				closedCasePage.getClosedCase(caseId).click();
+
+				viewClosedCasePage.hasManagedBy("Regions Group", "South West");
+			}));
 	});
 
+
+	it("Should show hint text", () => {
+		Logger.Log("Create a case");
```

Comment:
Need to create the case through the API, instead of the UI, otherwise it will slow the suite down.

Change the description to describe the overall process, in this case we are creating an SRMA for regions group